specification/botservice/resource-manager/Microsoft.BotService/stable/2021-03-01/botservice.json

```diff
@@ -0,0 +1,2640 @@
+        },
+        "customVoiceDeploymentId": {
+          "type": "string",
+          "description": "Custom speech model id (optional)."
+        },
+        "customSpeechModelId": {
+          "type": "string",
+          "description": "Custom voice deployment id (optional)."
+        },
+        "isDefaultBotForCogSvcAccount": {
+          "type": "boolean",
+          "description": "Make this a default bot for chosen cognitive service account."
+        }
+      },
+      "description": "The parameters to provide for the DirectLine Speech channel.",
+      "required": [
+        "cognitiveServicesSubscriptionId"
+      ]
+    },
+    "ChannelResponseList": {
+      "properties": {
+        "nextLink": {
+          "description": "The link used to get the next page of bot service channel resources.",
+          "type": "string"
+        },
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BotChannel"
+          },
+          "description": "Gets the list of bot service channel results and their properties."
+        }
+      },
+      "description": "The list of bot service channel operation response."
+    },
+    "WebChatSite": {
+      "properties": {
+        "siteId": {
+          "type": "string",
+          "description": "Site Id",
+          "x-ms-mutability": [
+            "read"
+          ],
+          "readOnly": true
+        },
+        "siteName": {
+          "type": "string",
+          "description": "Site name"
+        },
+        "key": {
+          "type": "string",
+          "description": "Primary key. Value only returned through POST to the action Channel List API, otherwise empty.",
+          "x-ms-mutability": [
+            "read"
+          ],
+          "readOnly": true
+        },
+        "key2": {
+          "type": "string",
+          "description": "Secondary key. Value only returned through POST to the action Channel List API, otherwise empty.",
+          "x-ms-mutability": [
+            "read"
+          ],
+          "readOnly": true
+        },
+        "isEnabled": {
+          "type": "boolean",
+          "description": "Whether this site is enabled for DirectLine channel"
+        },
+        "enablePreview": {
+          "type": "boolean",
+          "description": "Whether this site is enabled for preview versions of Webchat"
+        }
+      },
+      "required": [
+        "siteName",
+        "isEnabled",
+        "enablePreview"
+      ],
+      "description": "A site for the Webchat channel"
+    },
+    "DirectLineSite": {
+      "properties": {
+        "siteId": {
+          "x-ms-mutability": [
+            "read"
+          ],
+          "type": "string",
+          "description": "Site Id",
+          "readOnly": true
+        },
+        "siteName": {
+          "type": "string",
+          "description": "Site name"
+        },
+        "key": {
```

Comment:
same comment - it is an anti-pattern to return this for some operatiosn and not others, however, we should not break existing API